scripts/retrieveStorageSasUrl.js

```diff
@@ -0,0 +1,49 @@
+// Before you run the script, you need to
+// 1. Install the azure-storage package: `npm install azure-storage`
+// 2. Assign the storage connection string to the connectionString variable below
+
+const azure = require('azure-storage');
+
+let connectionString = "DefaultEndpointsProtocol=https;AccountName=...";
+
+function generateSasToken(container, connectionString) {
+  const blobName = "";
+  let blobService = azure.createBlobService(connectionString);
+
+  // Create a SAS token that expires in an hour
```

Comment:
Invalid comment. Expiry set to 60 days on line 18

Reply:
Thanks for catching this!